Impossible-Finance-Agents/Large-Deposit-Withdraw-On-Pairs/package.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "impossible-9",
   "version": "0.0.1",
-  "description": "Detect large add/remove Liquidity on Impossible Finance Pairs",
+  "description": "Detects large add/remove Liquidity on Impossible Finance Pairs",
+  "repository": "https://github.com/NethermindEth/Forta-Agents/tree/main/Impossible-Finance-Agents/Large-Deposit-Withdraw-On-Pairs",
```

Comment:
We need to include the `chainIds` in the `package.json` as well. However, we had already discussed this, so I think you were goin to add it anyway. 😎